lib/Threading/Internal/Thread.php

```diff
@@ -53,44 +53,44 @@ public function run() {
          * thread-safe objects.
          */
         $paths = [
-            \dirname(__DIR__, 5) . \DIRECTORY_SEPARATOR . 'autoload.php',
-            \dirname(__DIR__, 3) . \DIRECTORY_SEPARATOR . 'vendor' . \DIRECTORY_SEPARATOR . 'autoload.php',
+            \dirname(__DIR__, 3) . '/vendor/autoload.php',
+            \dirname(__DIR__, 5) . '/autoload.php',
         ];
-    
+
         $autoloadPath = null;
         foreach ($paths as $path) {
             if (\file_exists($path)) {
                 $autoloadPath = $path;
                 break;
             }
         }
-    
+
         if ($autoloadPath === null) {
             throw new \Error('Could not locate autoload.php');
         }
-    
-        require $autoloadPath;
-        
+
+        requireFile($autoloadPath);
```

Comment:
`(static function() use ($autoloadPath) { require $autoloadPath; })();` will just work as well and not need an extra function.